src/azul/service/hca_response_v5.py

```diff
@@ -475,7 +466,10 @@ def choose_entry(_term):
             term_object_params = {'term': choose_entry(term), 'count': term['doc_count']}
             if 'myProjectIds' in term:
                 term_object_params['projectId'] = [bucket['key'] for bucket in term['myProjectIds']['buckets']]
-            term_list.append(TermObj(**term_object_params))
+            if contents['myTerms']['meta']['path'][-1] == 'organism_age':
```

Comment:
Isn't there a more generic way to do this? One that doesn't depend on the cocrete field name?